changelog_unreleased/javascript/pr-9278.md

```diff
@@ -0,0 +1,61 @@
+#### Fix comments inside template (#9278 by @fisker)
+
+<!-- prettier-ignore -->
+```js
+// Input
+html`${
+      foo
+  /* comment */
+}`;
+html`
+${
+      foo
+  /* comment */
+}
+`;
+graphql`${
+      foo
+  /* comment */
+}`;
+css`${
+      foo
+  /* comment */
+}`;
+
+// Prettier stable
+html`${foo}`;
+/* comment */
+html`
+  ${foo}
+  /* comment */
+`;
+graphql`
+  ${foo}
+  /* comment */
+`;
+css`
+  ${foo}
+  /* comment */
+`;
+
+
+// Prettier master
+html`${foo
+/* comment */
+}`;
+html`
+  ${foo
+  /* comment */
+  }
+`;
+graphql`
+  ${foo
+  /* comment */
+  }
+`;
+css`
+  ${foo
+  /* comment */
+  }
+`;
```

Comment:
This is not very pretty, but I'm not sure how to improve it.

Reply:
Improved style https://github.com/prettier/prettier/pull/9278/commits/a35637f4d4e95217966365bb6bbd6361e6cfca59, but feel there is something wrong.